.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: c
+dist: trusty
```

Comment:
Yes. I am also not proposing this as a general distribution method, more as something to test the latest build without having to compile it. I could also use it for https://github.com/AppImage/AppImageHub